templates/developer.rackspace.com/_includes/share-bar.html

```diff
@@ -12,13 +12,13 @@ <h4>Share this information:</h4>
         </a>
 
         <a
-               href="https://twitter.com/intent/tweet?url={{ site.url }}{{ page.url }}&text={{ page.title }}&hashtags=rackspace" style="text-decoration: none!important"
+               href="https://twitter.com/home?status={{ site.url }}{{ page.url }}&text={{ page.title }}&hashtags=rackspace" style="text-decoration: none!important"
```

Comment:
I think we need to use twitter `intent` for this functionality. That link brought me to my Twitter home page without prompting me to share anything and not propagating the link from developer the web-site.